### PR TITLE
Follow changes in the storage spec.

### PR DESCRIPTION
**Pull request by mkruisselbrink**
This fixes various linking errors by updating to use newer terminology
from the storage spec. Also updates the text to be more consistent about
using "origin private" instead of "sandboxed".

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/native-file-system/pull/199.html" title="Last updated on Jul 14, 2020, 5:21 PM UTC (11320c2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/native-file-system/199/fa242dd...11320c2.html" title="Last updated on Jul 14, 2020, 5:21 PM UTC (11320c2)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1503,32 +1505,20 @@ these steps:
 
 </div>
 
-# Accessing the Sandboxed File System # {#sandboxed-filesystem}
+# Accessing the Origin Private File System # {#sandboxed-filesystem}
 
-## Sandboxed File System Concepts ## {#sandboxed-filesystem-concepts}
+The <dfn>origin private file system</dfn> is a [=storage endpoint=] whose
+<a spec=storage for="storage endpoint">identifier</a> is `"fileSystem"`,
```
Comment (inexorabletash) [inline index.bs:1511]: Can this just use the `[=storage  endpoint/identifier=]` syntax? Is the `spec=storage` necessary?

File: index.bs [hunk 2]
```diff
@@ -1538,10 +1528,11 @@ partial interface mixin WindowOrWorkerGlobalScope {
 </xmp>
 
 Advisement: In Chrome this functionality is currently exposed as `FileSystemDirectoryHandle.getSystemDirectory({type: "sandbox"})`.
+This new method is available as of Chrome 85.
```
Comment (inexorabletash) [inline index.bs:1531]: I assume this means that the new name/semantics is in 85+ ? Maybe reword "is currently exposed" to "was exposed as .... in Chrome 84 and earlier." ?

File: index.bs [hunk 3]
```diff
@@ -1550,16 +1541,26 @@ invoked, must run these steps:
...
 
-1. If |environment|'s [=environment settings object/origin=] is an [=opaque origin=],
+1. Let |map| be the result of running [=obtain a local storage bottle map=]
+   with |environment| and `"fileSystem"`. If this returns failure,
    return [=a promise rejected with=] a {{SecurityError}}.
 
-1. Let |storage| be |environment|'s [=environment settings object/origin=]'s [=storage unit=].
-1. Let |bucket| be |storage|'s [=bucket=].
+1. If |map|["root"] does not [=map/exist=]:
+  1. Set |map|["root"] to a new [=directory entry=].
```
Comment (inexorabletash) [inline index.bs:1549]: I think this would be slightly more readable with a temp var, e.g. 

1. Let _dir_ be a new directory entry.
1. Set _dir_'s foo to bar...
1. Set _dir_'s foo to bar...
1. Set |map|["root"] to _dir_

File: index.bs [hunk 4]
```diff
@@ -22,6 +22,8 @@ spec:webidl; type:dfn; text:resolve
 <pre class=anchors>
 urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
   type: dfn; text: realm; url: realm
+urlPrefix: https://storage.spec.whatwg.org/; spec: storage
+  type: dfn; text: storage; url: site-storage
```
Comment (inexorabletash) [inline index.bs:26]: What's this for? I don't see a `[=/storage=]` (etc) link

File: index.bs (hunk 1 above)
Comment (inexorabletash) [reply index.bs:1511]: I guess it would be if there's another spec defining storage endpoint at the moment? Which one?

Comment (mkruisselbrink) [reply index.bs:1511]: The spec=storage lets me link to it despite it not being exported...

File: index.bs (hunk 4 above)
Comment (mkruisselbrink) [reply index.bs:26]: This is a term storage used to provide, it is linked to from the privacy section `Directories where the user agent stores [=storage|website storage=]`.

File: index.bs (hunk 2 above)
Comment (mkruisselbrink) [reply index.bs:1531]: Changed it to "was previously exposed as" without explicitly saying when the old name went away, as it actually hasn't gone away yet, and will only be unavailable starting in M86/when we ship this outside of an origin trial. That hopefully is clear enough...

File: index.bs (hunk 3 above)
Comment (mkruisselbrink) [reply index.bs:1549]: Done

File: index.bs (hunk 4 above)
Comment (inexorabletash) [reply index.bs:26]: Ah, thanks. I missed that variant in my search!

File: index.bs (hunk 1 above)
Comment (inexorabletash) [reply index.bs:1511]: Thanks! (I add things to the `anchors` block in that case, whatever you prefer)

Comment (mkruisselbrink) [reply index.bs:1511]: The downside of the anchors block is that you no longer get an error if the thing you're trying to link to doesn't exist anymore.
